tests/helpers.py

```diff
@@ -189,7 +189,11 @@ def create_storage_class(
     sc_data['parameters']['csi.storage.k8s.io/provisioner-secret-name'] = secret_name
     sc_data['parameters']['csi.storage.k8s.io/provisioner-secret-namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
 
-    sc_data['parameters']['monitors'] = mons
+    if interface_type == constants.CEPHBLOCKPOOL:
+        sc_data['parameters']['clusterID'] = defaults.ROOK_CLUSTER_NAMESPACE
+    elif interface_type == constants.CEPHFILESYSTEM:
+        sc_data['parameters']['monitors'] = mons
```

Comment:
One is changing the clusterID and other the mons? Looks independent ?

Reply:
yes as per upstream repo https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml 
and 
https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/example/cephfs/storageclass.yaml